spec/controllers/plan_wizard_controller_spec.rb

```diff
@@ -47,6 +63,34 @@ def mocked_set_with_authorized_materials
         expect(flash[:alert]).to match(/not authori[sz]ed/)
       end
     end
+
+    context "when the product selected is not from SS" do
+      let(:catalogue) { create(:catalogue, url: 'not_ss_url') }
+      let(:product) { create(:product, catalogue: catalogue) }
+      let(:processes) { create_processes(product) }
+      let(:product_options) { processes.map { |pro| [pro.process_modules.first.id] } }
+
+      it 'should skip the data release strategy step' do
+        wp = create(:work_plan, owner_email: @user.email, original_set_uuid: SecureRandom.uuid, project_id: 123, product: product)
+        controller.instance_variable_set(:@work_plan, wp)
+
+        get :show, params: { work_plan_id: wp.id, id: 'data_release_strategy' }
+        expect(response.redirect_url).to match /dispatch*/
+      end
+    end
+    context "when the product selected is from SS" do
+      let(:catalogue) { create(:catalogue, url: Rails.configuration.ss_data_release_url) }
```

Comment:
Please could you change this to config.data_release_url (so it's consistent with the Projects app)